cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
@dnephin That is effort and time that isn't necessary for the feature. Forcing the design to happen in the process here will lead to a substandard design. Many packages in docker are exported and used in a single place, leading to a morass of exported interfaces without clear usage.